db/seeds/beers-attachments/East-Bank.svg

```diff
@@ -0,0 +1,148 @@
27,10.51102 -2.6339,2.87751 -7.36511,3.60492 -16.40701,2.52253 z m -45.47014,-28.62791 c 2.09255,-3.91204 3.40344,-14.03917 2.24676,-17.3572 -0.9044,-2.59438 -0.82974,-3.67535 0.38154,-5.52401 1.86754,-2.85022 1.93117,-5.77104 0.15778,-7.24281 -1.70908,-1.41842 -3.87435,0.8673 -5.68549,6.00177 -0.97789,2.77224 -1.0258,4.89466 -0.20318,9 0.92938,4.63812 0.81414,5.97039 -0.84332,9.74937 -1.05589,2.40741 -1.9198,4.99491 -1.9198,5.75 0,2.00482 4.75514,1.6991 5.86571,-0.37712 z m 12.92173,-10.11521 c -0.55522,-2.88304 -3.25053,-1.74795 -4.42901,1.86521 -0.62786,1.925 -0.89316,3.91136 -0.58954,4.41412 0.8154,1.35029 5.38643,-4.36908 5.01855,-6.27933 z m -28.96789,-3.37457 c -0.17575,-1.23189 -1.10705,-2.38988 -2.06955,-2.57333 -2.00595,-0.38232 -2.34145,1.82166 -0.55,3.61311 1.86216,1.86216 2.97079,1.42211 2.61955,-1.03978 z m 48.58285,-6.01022 c 2.28145,-18.47074 2.29692,-19.85058 0.26845,-23.95047 -2.29792,-4.64451 -4.20849,-5.69046 -6.39694,-3.50201 -1.30458,1.30457 -1.36856,2.05975 -0.3773,4.45287 0.8804,2.12547 0.90855,3.63188 0.10726,5.73946 -0.59756,1.57168 -0.80307,4.14785 -0.45671,5.72483 0.37167,1.69218 0.13062,3.46865 -0.58817,4.33474 -1.39024,1.67514 -0.70476,7.84705 1.11507,10.03981 0.64396,0.77592 2.21317,1.41077 3.48712,1.41077 1.97603,0 2.3934,-0.62431 2.84122,-4.25 z m 20.7476,-0.25 c 0,-0.5225 -0.48375,-1.11125 -1.075,-1.30833 -0.59125,-0.19709 -1.075,0.39166 -1.075,1.30833 0,0.91667 0.48375,1.50542 1.075,1.30833 0.59125,-0.19708 1.075,-0.78583 1.075,-1.30833 z m -60.15,-4.94098 c 0,-1.95514 -1.93837,-3.02065 -3.2,-1.75902 -0.56414,0.56415 -0.72339,1.51494 -0.35385,2.11287 0.98021,1.58602 3.55385,1.32977 3.55385,-0.35385 z m 3.18643,-10.54093 c 1.95977,-2.4202 2.44048,-7.5126 0.80462,-8.52362 -1.83008,-1.13105 -4.98811,0.59917 -6.5277,3.5764 -1.04058,2.01226 -2.27074,2.92913 -3.93001,2.92913 -2.78429,0 -3.97448,1.40803 -3.10514,3.6735 0.95564,2.49035 10.39272,1.26587 12.75823,-1.65541 z m 62.07558,-19.51809 c -0.32239,-6.7732 0.34369,-9.16363 2.72705,-9.78689 1.2068,-0.31558 2.01094,-1.36308 2.01094,-2.61949 0,-2.85886 -1.84102,-2.64197 -2.97892,0.35095 -0.82126,2.16005 -1.35949,2.37477 -4.62466,1.84491 l -3.69523,-0.59965 0.65869,4.96544 c 0.65334,4.92511 0.63018,4.97904 -2.85059,6.63891 -6.02079,2.87111 -4.45064,4.91827 3.45101,4.49945 l 5.5397,-0.29363 z m -31.76201,-5 c 0,-1.00833 -0.5625,-2.02083 -1.25,-2.25 -0.79167,-0.26389 -1.25,0.56111 -1.25,2.25 0,1.68889 0.45833,2.51389 1.25,2.25 0.6875,-0.22917 1.25,-1.24167 1.25,-2.25 z m -7.75,-2.75 c -0.22917,-0.6875 -1.01666,-1.25 -1.75,-1.25 -0.73333,0 -1.52083,0.5625 -1.75,1.25 -0.24786,0.74359 0.46111,1.25 1.75,1.25 1.28889,0 1.99786,-0.50641 1.75,-1.25 z m 30.23506,-21.79616 c -3.19684,-2.45625 -6.24831,-2.56355 -5.79703,-0.20384 0.21571,1.12791 1.57481,1.94404 3.82332,2.29585 4.55447,0.71261 5.00392,0.23623 1.97371,-2.09201 z m 8.01494,-5.51958 c 0,-1.84236 -2.35863,-4.43426 -4.03518,-4.43426 -2.39844,0 -2.47457,0.77925 -0.30983,3.17126 1.5489,1.7115 4.34501,2.52427 4.34501,1.263 
```

Comment:
Not sure why, but we don't seem to have a vector version of this one. The SVG is essentially a wrapper for an encoded raster image.